interfaces/cython/cantera/reactor.pyx

```diff
@@ -257,16 +255,16 @@ cdef class Reactor(ReactorBase):
     def add_sensitivity_species_enthalpy(self, k):
         """
         Specifies that the sensitivity of the state variables with respect to
-        species *k* should be computed. The reactor must be part of a network
+        species ``k`` should be computed. The reactor must be part of a network
         first.
         """
         self.reactor.addSensitivitySpeciesEnthalpy(self.thermo.species_index(k))
 
     def component_index(self, name):
         """
-        Returns the index of the component named *name* in the system. This
+        Returns the index of the component named ``name`` in the system. This
         determines the (relative) index of the component in the vector of
```

Comment:
Why is "relative" in parentheses here? Is it part of the sentence or not? Can we take it out of the parentheses and clarify, relative to what?